packages/hardhat-core/test/internal/hardhat-network/stack-traces/test-files/0_8/user-defined-value-types/c.sol

```diff
@@ -0,0 +1,12 @@
+pragma solidity ^0.8.8;
+
+import "./../../../../../../../console.sol";
```

Comment:
You can remove this now (and also the import in the `test.json`).

Reply:
Addressed in https://github.com/nomiclabs/hardhat/pull/2017/commits/ba4bfc18efd3c0fba58a98b9821fd42db3ddd9f7